.github/workflows/draft-release-notes-workflow.yml

```diff
@@ -16,6 +16,6 @@ jobs:
         with:
           config-name: draft-release-notes-config.yml
           tag: (None)
-          version: 2.4.0.0
+          version: 2.3.0.0
```

Comment:
I don't think we want to revert this, right? Peng just updated this to 2.4

Reply:
Thank you, fixed in 1d935e6c.